dev-packages/application-manager/src/generator/frontend-generator.ts

```diff
@@ -128,209 +128,229 @@ const { fork } = require('child_process');
 const { app, shell, BrowserWindow, ipcMain, Menu } = electron;
 
 const applicationName = \`${this.pck.props.frontend.config.applicationName}\`;
-
-const nativeKeymap = require('native-keymap');
-const Storage = require('electron-store');
-const electronStore = new Storage();
-
-let canPreventStop = true;
-const windows = [];
-
-app.on('before-quit', async event => {
-    if (canPreventStop) {
-        // Pause the stop.
-        event.preventDefault();
-        let preventStop = false;
-        // Ask all opened windows whether they want to prevent the \`close\` event or not.
-        for (const window of windows) {
-            if (!preventStop) {
-                window.webContents.send('prevent-stop-request');
-                const preventStopPerWindow = await new Promise((resolve) => {
-                    ipcMain.once('prevent-stop-response', (_, arg) => {
-                        if (!!arg && 'preventStop' in arg && typeof arg.preventStop === 'boolean') {
-                            resolve(arg.preventStop);
-                        }
-                    })
-                });
-                if (preventStopPerWindow) {
-                    preventStop = true;
+const wantSingleInstance = ${this.pck.props.backend.config.singleInstance === true ? 'true' : 'false'};
```

Comment:
[minor] `isSingleInstance` would be a better name and more consistent with other booleans.

Reply:
I've renamed this.